docs/architecture/gas/parameter_definition.md

```diff
@@ -10,7 +10,27 @@ starting from version 53. To see all parameter values for a specific version at
 once, check out the list of JSON snapshots generated in this directory:
 [core/primitives/src/runtime/snapshots](https://github.com/near/nearcore/blob/d0dc37bf81f7e7bde9c560403b085fae04108659/core/primitives/src/runtime/snapshots).
 
-<!-- TODO: Runtime Config Store -->
+## Using Parameters in Code
+
+The introduction on this page already hints at it. Parameter values are
+versioned. In other words, they can change between protocol versions. A nearcore
```

Comment:
I believe this to be the case and I wonder if we should specify that any changes to parameter values must happen via protocol upgrades.

Reply:
Yeah you are right. I though I already described it like you say. Now I rephrased slightly to make it clearer that parameter changes can only happen if we also change the protocol version.